frontend/packages/core/src/Assets/colors.tsx

```diff
@@ -0,0 +1,197 @@
+// COLORS FROM FIGMA
+// NOTE THAT THOSE WITH GRADIENTS ONLY HAVE THEIR BASES
+import type { ClutchPalette } from "./colortypes";
+
+export const LIGHT_MODE: ClutchPalette = {
+  NEUTRAL: {
+    5: "#F8F8F9", // PRIMARY BUTTON DISABLED / STROKE TERTIARY
```

Comment:
Looks great! One nit, could we move these line comments to the interfaces so they're referenced if someone creates a new palette?